docs/administration/Memory_management.md

```diff
@@ -27,8 +27,8 @@ Explanation:
 | mem_limit | 80% | The percentage of total memory that BE can use. If BE is deployed as a standalone, there is no need to configure it. If it is deployed with other services that consume more memory, it should be configured separately. |
 | disable_storage_page_cache | false | The boolean value to control if to disable PageCache. When PageCache is enabled, StarRocks caches the query results. PageCache can significantly improve the query performance when similar queries are repeated frequently. `true` indicates to disable PageCache. Use this item together with `storage_page_cache_limit`, you can accelerate query performance in scenarios with sufficient memory resources and much data scan. The value of this item has been changed from `true` to `false` since StarRocks v2.4. |
 | write_buffer_size | 104857600 |  The capacity limit of a single MemTable, exceeding which a disk swipe will be performed. |
-| load_process_max_memory_limit_bytes | 107374182400 | The total import memory limit `min(mem_limit * load_process_max_memory_limit_bytes, load_process_max_memory_limit_bytes)`. It is the actual available import memory threshold, reaching which a disk swipe will be triggered.  |
-| load_process_max_memory_limit_percent | 30 | The total import memory limit `min(mem_limit * load_process_max_memory_limit_percent, load_process_max_memory_limit_bytes)`. It is the actual available import memory threshold, reaching which the swipe will be triggered. |
+| load_process_max_memory_limit_bytes | 107374182400 | The maximum size limit of memory resources can be taken up by all load process on a BE node. Its value is the smaller one between `mem_limit * load_process_max_memory_limit_bytes` and `load_process_max_memory_limit_bytes`. It is the actual available import memory threshold, reaching which a disk swipe will be triggered.  |
```

Comment:
1. The upper limit of memory resources that can be taken up by all load processes on a BE node. 2. the formula is not correct, different from the cn version